docs/src/inputs/checkbox/devSandbox/devSandbox.jsx

```diff
@@ -253,26 +278,33 @@ export default class ElementsCheckbox extends React.Component {
                     </Highlighter>
 
                     {/* Label Click */}
-                    <Header size="large" style={{ marginTop: '55px' }} sub>
+                    <Heading
+                        anchorLink="label-click"
+                        variant="h2"
+                    >
                         Label Click
-                        <Header.Subheader>
-                            Sometimes you may want to disable a checkbox from being checked by clicking it's label definition.
-                        </Header.Subheader>
-                    </Header>
+                    </Heading>
 
+                    <Typography>
+                        Sometimes you may want to disable a checkbox from being checked by clicking it's label definition.
```

Comment:
Wrong "its" here (should not have an apostrophe) but non-blocking as it was like this and wasn't really changed as part of this PR.

Reply:
Fixed.

Reply:
Resolving.